torch/ao/quantization/fx/quantize_handler.py

```diff
@@ -144,25 +145,30 @@ def _get_pattern_to_quantize_handlers(backend_config: BackendConfig) -> Dict[Pat
 
 # TODO: remove this class, this is still exposed in torch.ao.quantization
 # but we should be able to break bc
+@deprecated("removed")
 class BinaryOpQuantizeHandler(QuantizeHandler):
```

Comment:
I don't think these are formally deprecated yet, so you shouldn't put the deprecated handler on them